.github/workflows/ci.yml

```diff
@@ -311,7 +311,6 @@ jobs:
         run: invoke ci-tests 'cloud' --up-services --verbose
 
   marker-tests:
-    needs: [unit-tests, static-analysis]
```

Comment:
Why is this changing?
Do we have enough workers now to handle this early fan-out?

Reply:
This is a temporary change that I plan on reverting before asking for reviews.